test/e2e/suites/import-gitops/import_gitops_test.go

```diff
@@ -1,139 +0,0 @@
-		return specs.CreateUsingGitOpsSpecInput{
-			E2EConfig:                 e2e.LoadE2EConfig(),
-			BootstrapClusterProxy:     bootstrapClusterProxy,
-			ClusterTemplate:           e2e.CAPIDockerKubeadm,
-			AdditionalTemplates:       [][]byte{e2e.CAPIKindnet},
-			ClusterName:               "clusterv1-docker-kubeadm",
-			ControlPlaneMachineCount:  ptr.To[int](1),
-			WorkerMachineCount:        ptr.To[int](1),
-			GitAddr:                   gitAddress,
-			SkipDeletionTest:          false,
-			LabelNamespace:            true,
-			TestClusterReimport:       true,
-			RancherServerURL:          hostName,
-			CAPIClusterCreateWaitName: "wait-rancher",
-			DeleteClusterWaitName:     "wait-controllers",
-		}
-	})
-})
-
-var _ = Describe("[AWS] [EKS] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.FullTestLabel), func() {
-	BeforeEach(func() {
-		komega.SetClient(bootstrapClusterProxy.GetClient())
-		komega.SetContext(ctx)
-	})
-
-	specs.CreateUsingGitOpsSpec(ctx, func() specs.CreateUsingGitOpsSpecInput {
-		return specs.CreateUsingGitOpsSpecInput{
-			E2EConfig:                 e2e.LoadE2EConfig(),
-			BootstrapClusterProxy:     bootstrapClusterProxy,
-			ClusterTemplate:           e2e.CAPIAwsEKSMMP,
-			ClusterName:               "clusterv1-eks",
-			ControlPlaneMachineCount:  ptr.To[int](1),
-			WorkerMachineCount:        ptr.To[int](1),
-			GitAddr:                   gitAddress,
-			SkipDeletionTest:          false,
-			LabelNamespace:            true,
-			RancherServerURL:          hostName,
-			CAPIClusterCreateWaitName: "wait-capa-create-cluster",
-			DeleteClusterWaitName:     "wait-eks-delete",
-		}
-	})
-})
-
-var _ = Describe("[vSphere] [Kubeadm] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.LocalTestLabel), func() {
-	BeforeEach(func() {
-		SetClient(bootstrapClusterProxy.GetClient())
-		SetContext(ctx)
-	})
-
-	specs.CreateUsingGitOpsSpec(ctx, func() specs.CreateUsingGitOpsSpecInput {
-		return specs.CreateUsingGitOpsSpecInput{
-			E2EConfig:                 e2e.LoadE2EConfig(),
-			BootstrapClusterProxy:     bootstrapClusterProxy,
-			ClusterTemplate:           e2e.CAPIvSphereKubeadm,
-			ClusterName:               "cluster-vsphere-kubeadm",
-			ControlPlaneMachineCount:  ptr.To[int](1),
-			WorkerMachineCount:        ptr.To[int](1),
-			GitAddr:                   gitAddress,
-			SkipDeletionTest:          false,
-			LabelNamespace:            true,
-			RancherServerURL:          hostName,
-			CAPIClusterCreateWaitName: "wait-capv-create-cluster",
-			DeleteClusterWaitName:     "wait-vsphere-delete",
-			AdditionalTemplateVariables: map[string]string{
-				"NAMESPACE":             "default",
-				"VIP_NETWORK_INTERFACE": "",
-			},
-		}
-	})
-})
-
-var _ = Describe("[vSphere] [RKE2] Create and delete CAPI cluster functionality should work with namespace auto-import", Label(e2e.LocalTestLabel), func() {
```

Comment:
Just noticed this - @alexander-demicev shouldn’t be these tests re-added into v3 suite?